consensus/common_test.go

```diff
@@ -37,6 +37,23 @@ func ensureDir(dir string, mode os.FileMode) {
 	}
 }
 
+type NodeConfig struct {
+	cfg.Config `mapstructure:",squash"`
+	P2P        *p2p.Config   `mapstructure:"p2p"`
+	Mempool    *mempl.Config `mapstructure:"mempool"`
+	Consensus  *Config       `mapstructure:"consensus"`
+}
+
+// TODO: This is the same as NodeConfig. Should we move the configs to types (?)
```

Comment:
Note this is a duplicate of node.Config. Not sure if we can do better but it kind of bugs me. One solution is to put all the configs together where any package can import them, like tendermint/tendermint/config, but it does seem better to have a config in its own package. Maybe this is just the price and its fine?